bash/jhail

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+source "$(dirname $0)"/setup_env
+
+jupyter nbextension list 2>&1 | grep 'jupyter_spark *enabled'
```

Comment:
I could nest the warnings so you only get the first one that fails. I figured if I was using it that I'd want to know all the things that are wrong so I can fix them in one fell swoop.

This all feels too much of a hack. I'll remove it all and assume the users who care will install jupyter_spark.